www/components/ModuleListSidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Box } from "@welcome-ui/box";
+import { Text } from "@welcome-ui/text";
+import { Link } from "@welcome-ui/link";
+import styled from "styled-components";
+
+import { Pkg } from "../lib/docs";
+
+const ModuleLink = styled(Link)<{ active: boolean }>`
+  padding-left: ${(props) => props.theme.space.xs};
+  padding-right: ${(props) => props.theme.space.md};
+  width: 100%;
+  ${(props) =>
+    props.active &&
+    `
+    background-color: ${props.theme.colors.light["500"]};
+    font-weight: bold;
+    color: ${props.theme.colors.dark["900"]};
+  `}
```

Comment:
jesus... is this The Way?

Reply:
The alternative is one component with the shared styles, one component wrapping that with the active styles, and a third component switching between them. This felt cleaner (and it's *significantly* cleaner with `@xstyled` syntax)